src/Application/Common/Behaviours/LoggingBehaviour.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Application.Common.Interfaces;
+﻿using CleanArchitecture.Application.Common.Interfaces;
```

Comment:
```suggestion
﻿using SSW.CleanArchitecture.Application.Common.Interfaces;
```

This should flow through the whole template